.circleci/config.yml

```
@@ -15,6 +15,18 @@ workflows:
             tags:
               only: /^v.*/
 
+      - architect/push-to-docker:
+          name: push-standup-to-quay
```

```suggestion
          name: push-apptestctl-to-quay
```

It also needs the architect context to be able to access the env vars.